aten/src/ATen/native/TensorShape.cpp

```diff
@@ -2170,6 +2170,12 @@ Tensor alias(const Tensor& self) {
     return alias_with_sizes_and_strides(self, self.sizes(), self.strides());
 }
 
+Tensor detach(const Tensor& self) {
```

Comment:
What is the low level implication of moving detach here without any modification of native_functions.yaml?

Reply:
doesn't matter. I just put it here so that it's close to alias.

Reply:
Ho ok :p